Makefile

```diff
@@ -0,0 +1,125 @@
+# Two main optional variables are required:
+# - $(V): set or unset. If set the verbose output will be printed.
+#         cargo verbosity is set to default. To override `cargo build`
+#         behaviour please refer to $(CARGO_COMMON_OPTS) for common
+#         options and $(CARGO_BUILD_OPTS) for build options.
+#         For further `cargo` customization please refer to
+#         configuration environment variables
+#         (https://doc.rust-lang.org/cargo/reference/environment-variables.html).
+# - $(NOVENDOR): avoid to self detect and consume the vendored headers
+#                shipped with libbpf-sys
+#
+ROOT_DIR := $(dir $(abspath $(lastword $(MAKEFILE_LIST))))
+RELEASE_VERSION ?= 1.3.0
+RELEASE_NAME ?= bredele
+
+export LLC := llc
+export CLANG := clang
+export OBJCOPY := llvm-objcopy
+
+PRINT=echo
+
+ifeq ($(V),)
+	Q=@
+	MAKE += -s
+	suppress=$(Q)>/dev/null 2>&1 
+define out_console
+	$(PRINT) -e "\033[33m [$(1)] $(2)\033[0m"
```

Comment:
Nit: we can use a tab to have the output better aligned. Something like `"\033[33m $(1)\t$(2)\033[0m"`.

Also not sure about using a color. It's nice, but depending the ones' terminal bg, this can be unreadable.

Reply:
I agree. I would happily avoid adding any kind of detection or similar things. The best option is to remove it.